sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/epkversion/BootstrapperImpl.java

```diff
@@ -21,20 +22,28 @@ class BootstrapperImpl implements Bootstrapper {
     private final Logger logger = LoggerFactory.getLogger(BootstrapperImpl.class);
     private final PartitionSynchronizer synchronizer;
     private final LeaseStore leaseStore;
+    private final LeaseStoreManager pkVersionLeaseStoreManager;
```

Comment:
I know the term "pkVersion" comes from a previous PR - so non-blocking. I got confused by this prefix - pk - assuming stands for partition key - what is meant here is PKRangeId (PartitionKeyRangeId) - i was wondering why you want a logical partition based lease store - until I realized pk stands for partitioneyrangeid not partitionkey. First thought it is a typo missing the e of epk :-) - anyway like mentioned non-blocking but IMO not very intuitive.

Reply:
make sense, maybe pkRangeIdVersion?

Reply:
much better IMO